packages/amplify-graphql-transformer-core/src/cdk-compat/stack-synthesizer.ts

```diff
@@ -12,18 +13,12 @@ export class TransformerStackSythesizer extends LegacyStackSynthesizer {
   private readonly mapingTemplateAssets: Map<string, string> = new Map();
   private _deploymentBucket?: CfnParameter;
   private _deploymentRootKey?: CfnParameter;
-  private boundStack?: Stack;
-
-  /**
-   * bind
-   */
-  public bind(stack: Stack): void {
-    this.boundStack = stack;
-    super.bind(stack);
-  }
 
   /**
    * synthesizeStackTemplate
+   *
+   * This method has been deprecated by cdk and is not used in runtime.
+   * It got replaced by synthesizeTemplate.
```

Comment:
/**
 * @deprecated Use synthesizeTemplate instead
 */

Reply:
I added deprecation marker, but I keep wording. Technically we're not "using it" but CDK calls into it at runtime.